release_list.yml

```diff
@@ -2,8 +2,6 @@
 # List releases in the order they should appear in the documentation
 # version menu.  Default at the top of the list.
 releases:
-  - 2.7.1
-  - 2.7.0
+  - 2.7.3
```

Comment:
do we also need a new entry in `upgrade/Notable_Changes.md`?

Reply:
Updated the Notable_Changes.md file.